docs/quick_start.md

```diff
@@ -0,0 +1,153 @@
+2016-03-01 20:54:38,357 - [INFO] - container.py - Skipping pulling Docker image: projectatomic/helloapache
+2016-03-01 20:54:38,358 - [INFO] - container.py - Extracting nulecule data from image: projectatomic/helloapache to /host/var/lib/atomicapp/projectatomic-helloapache-a68057164f09
+6eedd332f9938c7b4bacca694fdc77309ca5b43aabb05a1cb644ff8a0b713012
+2016-03-01 20:54:38,558 - [WARNING] - plugin.py - Configuration option 'providerconfig' not found
+2016-03-01 20:54:38,558 - [WARNING] - plugin.py - Configuration option 'providerconfig' not found
+2016-03-01 20:54:38,602 - [INFO] - docker.py - Deploying to provider: Docker
+a98d9a3305496803c38a90a9ef65c52030dc23dae4b04f36ce167ff98335395f
+
+Your application resides in /var/lib/atomicapp/projectatomic-helloapache-a68057164f09
+Please use this directory for managing your application
+```
+
+__Runnning Apache on Kubernetes:__
+```sh
+▶ sudo atomic run projectatomic/helloapache --provider=kubernetes
+docker run -it --rm  --privileged -v /home/wikus:/atomicapp -v /run:/run -v /:/host --net=host --name helloapache -e NAME=helloapache -e IMAGE=projectatomic/helloapache projectatomic/helloapache  run  --provider=kubernetes
+docker run -it --rm  --privileged -v /home/wikus:/atomicapp -v /run:/run -v /:/host --net=host --name helloapache -e NAME=helloapache -e IMAGE=projectatomic/helloapache projectatomic/helloapache  run  --provider=kubernetes
+2016-03-01 20:58:03,396 - [INFO] - main.py - Action/Mode Selected is: run
+2016-03-01 20:58:03,397 - [INFO] - base.py - Unpacking image: projectatomic/helloapache to /host/var/lib/atomicapp/projectatomic-helloapache-89e975ea7438
+2016-03-01 20:58:04,153 - [INFO] - container.py - Skipping pulling Docker image: projectatomic/helloapache
+2016-03-01 20:58:04,153 - [INFO] - container.py - Extracting nulecule data from image: projectatomic/helloapache to /host/var/lib/atomicapp/projectatomic-helloapache-89e975ea7438
+c85cbb2d28857f2b283e23a72a70e077daceeb2b72f6964605af6f7efa8fbc2f
+2016-03-01 20:58:04,387 - [WARNING] - plugin.py - Configuration option 'providerconfig' not found
+2016-03-01 20:58:04,388 - [WARNING] - plugin.py - Configuration option 'providerconfig' not found
+2016-03-01 20:58:04,388 - [INFO] - kubernetes.py - Using namespace default
+2016-03-01 20:58:04,388 - [INFO] - kubernetes.py - trying kubectl at /host/usr/bin/kubectl
+2016-03-01 20:58:04,388 - [INFO] - kubernetes.py - trying kubectl at /host/usr/local/bin/kubectl
+2016-03-01 20:58:04,388 - [INFO] - kubernetes.py - found kubectl at /host/usr/local/bin/kubectl
+2016-03-01 20:58:04,388 - [INFO] - kubernetes.py - Deploying to Kubernetes
+
+Your application resides in /var/lib/atomicapp/projectatomic-helloapache-89e975ea7438
+Please use this directory for managing your application
+```
+
+__Stopping Apache on Kubernetes:__
+```sh
+▶ sudo atomic stop projectatomic/helloapache /var/lib/atomicapp/projectatomic-helloapache-89e975ea7438 --provider=kubernetes
```

Comment:
we don't need to provider `--provider=kubernetes` in this command - it is captured in the `answers.conf.gen` file